cluster/examples/kubernetes/ceph/object-bucket-claim-delete.yaml

```diff
@@ -11,3 +11,7 @@ spec:
   #bucketName: 
   generateBucketName: ceph-bkt
   storageClassName: rook-ceph-delete-bucket
+  additionalConfig:
```

Comment:
Perhaps all of the example yamls using `additionalConfig` keys should be commented out, meaning the default examples show that this can be done but do not apply quotas.

Reply:
will address in next PR

Reply:
Done